Student/Widgets/AnnouncementsWidget/Model/AnnouncementItem.swift

```diff
@@ -60,3 +60,29 @@ struct AnnouncementItem: Identifiable, Equatable {
         self.avatar = nil
     }
 }
+
+extension AnnouncementItem {
```

Comment:
```suggestion
extension Array where Element == AnnouncementItem {
       static let publicPreviewItems: [Self] { ... }
```

This way, you can call directly on that `AnnouncementsEntry` initializer:
```swift
      .init(announcements: .publicPreviewItems)
```

Reply:
Done.